src/__tests__/themeContext.spec.ts

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import inquirer from 'inquirer';
+import { URLS } from '../lib/api/services/url';
+import mockFunction from './helper';
+const appConfig = require('./fixtures/appConfig.json');
+const oauthData = require('./fixtures/oauthData.json');
+const data = require('./fixtures/email-login.json');
+import configStore from '../lib/Config';
+import { decodeBase64 } from '../helper/utils';
+import fs from 'fs-extra';
+import path from 'path';
+import { init } from '../fdk';
+import rimraf from 'rimraf';
+import { readFile } from '../helper/file.utils';
+import CommandError from '../lib/CommandError';
+
+jest.mock('inquirer');
+let program;
+
+afterEach(() => {
+    configStore.clear();
+});
+
+afterAll(() => {
+    let filePath = path.join(process.cwd(), './.fdk/context.json');
+    try {
+        rimraf.sync(filePath);
+    } catch (err) {
+        console.error(`Error while deleting ${filePath}.`);
+    }
+});
+async function login() {
+    const inquirerMock = mockFunction(inquirer.prompt);
+    inquirerMock.mockResolvedValue({ password: '1234567' });
+    await program.parseAsync(['ts-node', './src/fdk.ts', 'login', '-e', 'anuragpandey@gofynd.com']);
+}
+
+describe('Theme Context Commands', () => {
+    beforeEach(async () => {
+        await program.parseAsync(['ts-node', './src/fdk.ts', 'env', 'set', '-n', 'fyndx0']);
+        program.commands.forEach(command => {
+            command._optionValues = {};
+        });
+    });
+
+    beforeAll(async () => {
+        program = await init('fdk');
+        const mock = new MockAdapter(axios);
+        mock.onPost(`${URLS.LOGIN_USER()}`).reply(200, data, {
+            'set-cookie': [{ Name: 'Anurag Pandey' }],
```

Comment:
remove name Anurag better you check for actual cookie value and @anuragpandey1115